dts/arm/atmel/same70q21b.dtsi

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2018 Linaro Limited
+ * Copyright (c) 2020 Stephanos Ioannidis <root@stephanos.io>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <atmel/same70q21.dtsi>
```

Comment:
I wanted to avoid creating duplicate code, but I agree that this approach is rather unconventional and can potentially be confusing.

I will just copy and paste as you suggested since the amount of duplicate code here isn't that much.